proto/api_v3/query_service.proto

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) 2019 The Jaeger Authors.
+// Copyright (c) 2018 Uber Technologies, Inc.
```

Comment:
`Copyright (c) 2021 The Jaeger Authors.`

Or perhaps we should start omitting the year [like in OTEL collector](https://github.com/open-telemetry/opentelemetry-collector/blob/c8e28f264c9b293a9bf2b0e7a78c54a962c46650/processor/memorylimiter/internal/cgroups/doc.go#L1)?